client/src/lib/components/sandbox/play/MainGamePage.svelte

```diff
@@ -30,8 +30,8 @@
 	{#if loaded}
 		{#key currentGame}
 			<div>It's your turn: {truncateMinaPublicKey(currentPlayer())}</div>
-			<PhaseInput game={currentGame} currentPlayer={currentPlayer()} {rerender} />
-			<Arena game={currentGame} />
+			<div>Phase: {currentGame.currentPhase.name}</div>
```

Comment:
Obviously the intention is to replace all of the phase inputs with mouse events but for now this makes it impossible to finish a game.  Could you short circuit the movement phase only in `client/src/lib/components/sandbox/play/phase-input/PhaseInput.svelte`?